lib/SILOptimizer/IPO/UsePrespecialized.cpp

```diff
@@ -88,20 +88,19 @@ bool UsePrespecialized::replaceByPrespecialized(SILFunction &F) {
     if (Subs.empty())
       continue;
 
-    ReabstractionInfo ReInfo(ReferencedF, Subs);
-
-    auto SpecType = ReInfo.getSpecializedType();
-    if (!SpecType)
+    // Bail if any generic type parameters are unbound.
+    if (hasUnboundGenericTypes(Subs))
```

Comment:
We can remove this limitation once we support public partial specializations exposed by other modules. I'll add a TODO comment about this.